adapters/repos/db/shard_snapshot.go

```diff
@@ -89,18 +90,18 @@ func (s *Shard) readBackupMetadata(d *backup.ShardDescriptor) (err error) {
 	if d.DocIDCounter, err = os.ReadFile(fpath); err != nil {
 		return fmt.Errorf("read shard doc-id-counter %s: %w", fpath, err)
 	}
-	d.DocIDCounterPath = fpath
+	d.DocIDCounterPath = path.Base(fpath)
```

Comment:
Could you please explain why are we taking path.Base().
Isn’t the base() just returning the last element of the path